### ISSUE TITLE
Leaves use reflection and appear glassy (w/ iris sildurs shaders)

### ISSUE THREAD

**Issue by JSJBDEV** [raltsmc/desolation#23]
![image](https://user-images.githubusercontent.com/16166991/112704395-5fe2ca80-8e92-11eb-935b-33807e49701a.png)
This isn't a really high priority thing, but I assume what is happening here is your leaves dont use the same render system as vanilla, thus with shaders they end up like this? I havent tested with optifine on forge or fabric so it may simply be a bug in iris, if you think that is the case then Ill go pester them instead 👍 
thank you for your time!

**Comment by raltsmc** [comment]
Yeah, the leaves use translucent rendering at the moment which is probably causing issues with certain shaders. I had a similar issue when testing it with Optifabric earlier. It'll be fixed in the next update :)
